terraform/modules/aws/rds/README.md

```diff
@@ -0,0 +1,65 @@
+## Requirements
```

Comment:
Are these files auto-generated somehow or are you manually creating them and keeping them in sync with `variables.tf` etc.?

Reply:
These were auto-generated using [terraform-docs](https://github.com/terraform-docs/terraform-docs). We do not use it on project level (yet) didn't want to bloat this PR introducing it -- as it is unrelated to this change set.